compiler/qsc/src/target.rs

```diff
@@ -39,6 +47,7 @@ impl FromStr for Profile {
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
```

Comment:
Maybe `s.to_lowercase().as_str()` and just keep the latter patterns for matching (then any mix of casing is handled).